SPECS/kata-containers/kata-containers.spec

```diff
@@ -143,12 +133,13 @@ install -m 0755 -D -t %{buildroot}%{katauvmdir} %{SOURCE3}
 install -m 0644 -D -t %{buildroot}%{katauvmdir} VERSION
 install -m 0644 -D -t %{buildroot}%{katauvmdir} versions.yaml
 install -D -m 0644 ci/install_yq.sh %{buildroot}%{katauvmdir}/ci/install_yq.sh
-sed -i 's#distro_config_dir="${script_dir}/${distro}#distro_config_dir="${script_dir}/cbl-mariner#g' tools/osbuilder/rootfs-builder/rootfs.sh
+sed --follow-symlinks -i 's#distro_config_dir="${script_dir}/${distro}#distro_config_dir="${script_dir}/cbl-mariner#g' tools/osbuilder/rootfs-builder/rootfs.sh
 
 pushd src/runtime
 %make_install %{runtime_make_vars}
-sed -i -e "s|image = .*$|initrd = \"%{katainitrddir}/kata-containers-initrd.img\"|" %{buildroot}%{kataconfigdir}/configuration.toml
-sed -i -e "s|kernel = .*$|kernel = \"%{kataclhdir}/vmlinux.bin\"|" %{buildroot}%{kataconfigdir}/configuration.toml
+# Ensure sed doesn't replace the configuration.toml symlink by a regular file.
+sed --follow-symlinks -i -e "s|image = .*$|initrd = \"%{katainitrddir}/kata-containers-initrd.img\"|" %{buildroot}%{kataconfigdir}/configuration.toml
+sed --follow-symlinks -i -e "s|kernel = .*$|kernel = \"%{kataclhdir}/vmlinux.bin\"|" %{buildroot}%{kataconfigdir}/configuration.toml
 popd
 
 pushd src/agent
```

Comment:
$files section:
do we need kata-monitor, kata-runtime, kata-collect-data.sh? In addition, these seem to be placed twice?

further, do we need to copy all configuration*.toml files?

Reply:
I'll leave these in for now